### PR TITLE
Add static html docs that contains list of icons

### PR DESCRIPTION
**Pull request by unasuke**
generated by icomoon.io

publish with github pages
https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/

### CODE REVIEW

File: tmix-icon-rails.gemspec [hunk 1]
```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.description   = %q{Web font for tmix}
   spec.homepage      = "https://github.com/spice-life/tmix-icon-rails"
 
-  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
+  spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(docs|test|spec|features)/}) }
```
Comment (unasuke) [inline tmix-icon-rails.gemspec:16]: exclude doc files from gemfile

**Comment by asonas** [comment]
How did you generate the files?
If there is a (useful) script, please add to the repository.

**Comment by asonas** [comment]
> generated by icomoon.io

Ahh okay.

**Comment by unasuke** [comment]
Thanks. 👍

**Pull request merged**